lib/index.es6

```diff
@@ -0,0 +1,34 @@
+import path from "path";
+import fs from "fs";
+
+import {sync as mkdirp} from "mkdirp";
+
+import compile from "./compile";
+import {entries} from "./util";
+
+export default function Interlock (options) {
```

Comment:
Why not use a class here? You're manually assigning to the prototype just like it would.